packages/core/src/State.ts

```diff
@@ -297,17 +299,22 @@ export class State<
    * Whether the current state value is a subset of the given parent state value.
    * @param parentStateValue
    */
-  public matches<TSV extends TTypestate['value']>(
-    parentStateValue: TSV
+  public matches<TSV extends TTypestate['value'] = never>(
```

Comment:
What's that `= never` here for? 🤔 it looks a little bit fishy

Reply:
Also in accidentally, good eyes